products/vmware_cb_response.py

```diff
@@ -59,14 +68,18 @@ def process_search(self, tag: Tag, base_query: dict, query: str) -> None:
             for proc in self._conn.select(Process).where(query):
                 result = Result(proc.hostname.lower(), proc.username.lower(), proc.path, proc.cmdline,
                                 (proc.start, proc.id))
+                raw_results.append(proc)
                 results.add(result)
+                
 
                 if self._limit > 0 and len(results)+1 > self._limit:
                         break
                 
         except KeyboardInterrupt:
             self._echo("Caught CTRL-C. Returning what we have . . .")
-
+        
+        if self._raw: return raw_results
```

Comment:
If the previous code change is implemented, this will need to be edited as well
```suggestion
        if self._raw: return results
```

Reply:
Can't do, each proc is a dict that will not be accepted by the set.